Doppler.AccountPlans.Test/CalculateUpgradeCostTest.cs

```diff
@@ -251,7 +282,7 @@ public void CalculateUpgradeCostHelper_Annually_method_should_return_correct_val
             var result = CalculateUpgradeCostHelper.CalculatePlanAmountDetails(newPlan, currentDiscountPlan, currentPlan, dateTimeProviderMock.Object.Now);
 
             // Assert
-            Assert.Equal(18, result.Total);
+            Assert.Equal(29, result.Total);
```

Comment:
Why did these tests not failed before this change?

Reply:
Because we found some problems in the way the total was being calculated and we made some changes to it that had an impact on the total.

Reply:
So, rethinking the previous code was a good idea. I glad.